docs/RESOURCES.md

```diff
@@ -1,2 +1,6 @@
 # Resources
 
+- [ARM64 reference card](https://courses.cs.washington.edu/courses/cse469/19wi/arm64.pdf)
+- [Official ARM documentation](https://developer.arm.com/documentation)
+- [The GAS Manual](https://sourceware.org/binutils/docs/as/)
+- [The GAS AArch64 Specifics Section](https://sourceware.org/binutils/docs/as/AArch64_002dDependent.html#AArch64_002dDependent)
```

Comment:
Reference links, as per [the Exercism markdown guide][1].

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md